### ISSUE TITLE
Video for generating metadata w/ pydantic

### ISSUE THREAD

**Issue by dbirman** [AllenNeuralDynamics/aind_data_outreach#36]
We need a video that introduces people to aind-data-schema and how to work with it and pydantic

Some things we might put in the video:

- Overview of aind-data-schema and how to navigate the schema
- Quick pointer to the documentation
- Example of writing a schema file while jumping back and forth to the references
- Setting up a virtual environment w/ vscode to be able to click-through into classes
- Ensuring that class validates and writes to the standard file properly

**Comment by dbirman** [comment]
Met on 11/5 to demo metadata and discuss requirements. Saskia will follow-up with rig example updates to add position information. Su-Yee will try building a rig and come back w/ any remaining questions.

Dan/Su-Yee will then write a script together

**Comment by saskiad** [comment]
Should we time this to the schema v2.0 release?

**Comment by dbirman** [comment]
> Should we time this to the schema v2.0 release?

Yeah probably especially if we end up merging things like instruments/rig. Otherwise that will be an obvious point of confusion

**Comment by saskiad** [comment]
@leesuyee @dbirman Just want to bring this back up - might be good to start thinking/working on this tutorial (and think about other tutorial material that we might want to have for 2.0)

**Comment by dbirman** [comment]
> [@leesuyee](https://github.com/leesuyee) [@dbirman](https://github.com/dbirman) Just want to bring this back up - might be good to start thinking/working on this tutorial (and think about other tutorial material that we might want to have for 2.0)

Let's meet and discuss this after the sci comp meeting next week?
